frontend/src/metabase/admin/settings/components/EmbeddingSettings/EmbeddingSettings.tsx

```diff
@@ -41,16 +43,17 @@ export function EmbeddingSettings({
               display_name: t`Embedding`,
             }}
           />
-          <SetByEnvVarWrapper setting={enableEmbeddingSetting}>
-            <EmbeddingSwitchWidget
```

Comment:
we don't need the embedding toggle anymore since it'll be replaced by invididual toggles instead.